index.html

```diff
@@ -30,279 +33,326 @@
+      <h1 class="primary-text">I’m Ankit kush. Glad to see you!</h1>
+      <p class="paragraph">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact me.
       </p>
-      <a href="#" class="connect about-connect">Let's connect</a>
-      <ul class="social about-social">
+      <h3>
+        <a class="connect" href="https://www.linkedin.com/in/ankit26k/">
+          LET'S CONNECT</a
+        >
+      </h3>
+
+      <ul class="social">
         <li>
           <a href="https://twitter.com/ankit26k">
-            <img class="icon" src="images/twitter.png" alt="twitter" /></a>
+            <img class="icon" src="images/twitter.png" alt="twitter"
+          /></a>
         </li>
 
         <li>
           <a href="https://www.linkedin.com/in/ankit26k/">
-            <img class="icon" src="images/Linkedin.png" alt="linkedin" /></a>
+            <img class="icon" src="images/Linkedin.png" alt="linkedin"
+          /></a>
         </li>
 
         <li>
           <a href="https://twitter.com/ankit26k">
-            <img class="icon meta" src="images/Vector1.png" alt="meta" /></a>
+            <img class="icon meta" src="images/Vector1.png" alt="meta"
+          /></a>
         </li>
 
         <li>
           <a href="https://github.com/ankitt26">
-            <img class="icon" src="./images/vector2.png" alt="github" /></a>
+            <img class="icon" src="images/vector2.png" alt="github"
+          /></a>
         </li>
 
         <li>
           <a href="https://www.instagram.com/ankit16t/">
-            <img class="icon" src="./images/vector3.png" alt="twofin" /></a>
+            <img class="icon" src="images/vector3.png" alt="twofin"
+          /></a>
         </li>
       </ul>
-      <button class="about-btn btn_btn">Get my resume</button>
     </section>
 
-    <!-- section 2 -->
+    <!-- Work section !-->
+    <section class="works" id="portfolio">
+      <div class="cards" id="one1">
+        <div class="block1">
+          <img
+            class="card-img"
+            src="images/Snapshoot.png"
+            alt="tonic project"
+          />
+        </div>
+        <div class="block2">
+          <div class="box1">
+            <h2 class="title1">Tonic</h2>
+            <ul class="frame1">
+              <li class="A Li">CANOPY</li>
+              <li class="BC Li">
+                <i class="bi bi-circle-fill icon"></i> Back End dev
+              </li>
+              <li class="BC Li"><i class="bi bi-circle-fill icon"></i> 2019</li>
+            </ul>
+          </div>
 
-    <section class="section2">
-      <ul class="section2-ul">
-        <li class="section2-li">
-          <h2 class="section2-h2">Language</h2>
-          <i class="bi bi-chevron-down icon2"></i>
-        </li>
+          <div class="box2">
+            <p class="card-para">
```

Comment:
- [ ] Two of the main requirements of this milestone are: 
1) to create all the projects cards dynamically 
2) Then, to build popups for each card dynamically also.
To fulfill these both requirements, we make use of the information we stored in the array of objects in JS file. Right now, you are creating popups dynamically from that array but you are not creating the projects cards dynamically so please consider removing everything inside the `<section class="works" id="portfolio">` and create all the cards dynamically from the same array of objects.
This way adding projects in the future will be easy, all you need to do would be to add another object in that array.
-  You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.